react-function-components/src/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function CustomButton(props) {
+  return <button>Click Me!</button>;
+}
+
+ReactDOM.render(CustomButton(), document.getElementById('root'));
```

Comment:
When writing React code, we generally never call our own components. Instead create a react element.

```jsx
ReactDOM.render(<CustomButton />, root);

```